src/views/Activate/ActivateCode.js

```diff
@@ -134,20 +134,17 @@ export default function ActivateCode() {
           as={ForwardButton}
           className="mt4"
           disabled={!pass || deriving}
-          accessory={deriving ? <Blinky /> : null}
+          accessory={deriving ? <Blinky /> : undefined}
```

Comment:
Does react care about this difference when it comes to rendering?

Reply:
react doesn't, but javascript does — in the following syntax

```
<Component accessory="default" {...props} />
```

passing `undefined` gives the component "default" and passing `null` gives the component `null`

(the idea being that `undefined` is specifically the case where "nothing at all" was passed